doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -64,15 +64,20 @@ Common
 Edge Impulse
 ------------
 
+* Added power manager to the :ref:`nrf_machine_learning_app` application.
 * Updated information about custom build types.
 
 Common Application Framework (CAF)
 ----------------------------------
 
+Added:
+
+* Added :ref:`caf_buttons_pm_keep_alive`.
 
 Updated:
 
 * The power management support in modules is now enabled by default when the :kconfig:`CONFIG_CAF_PM_EVENTS` Kconfig option is enabled.
+* Mark the dependency of :ref:`caf_power_manager` on :kconfig:`PM_POLICY_APP` that is required for application with :ref:`caf_power_manager` to link.
```

Comment:
```suggestion
* The :ref:`caf_power_manager` now has a dependency on :kconfig:`CONFIG_PM_POLICY_APP`, which is required by the application that is using the :ref:`caf_power_manager` to link.
```
What does "to link" mean at the end?

Reply:
@greg-fer
To link the code - this is the process after compilation that connects all the object files and libraries in one binary code.